src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -1167,3 +1168,96 @@ def forward(
             hidden_states=transformer_outputs.hidden_states,
             attentions=transformer_outputs.attentions,
         )
+
+
+@add_start_docstrings(
+    """
+    The BLOOM Model transformer with a span classification head on top for extractive question-answering tasks like
+    SQuAD (a linear layers on top of the hidden-states output to compute `span start logits` and `span end logits`).
+    """,
+    BLOOM_START_DOCSTRING,
+)
+class BloomForQuestionAnswering(BloomPreTrainedModel):
+    _keys_to_ignore_on_load_missing = [r"h.*.self_attention.scale_mask_softmax.causal_mask", r"lm_head.weight"]
+
+    def __init__(self, config):
+        super().__init__(config)
+        self.num_labels = config.num_labels
```

Comment:
It's a stupid question but it seems that for Question Answering the number of labels is always 2 (`start_logits`, `end_logits`), shall we hardcode it to 2? Maybe I am missing something here 🤔

Reply:
Perfect thanks, now it makes sense yes ! Then we should also modify it for GPTJ maybe? Happy to open a PR later for that